tokio/tests/rt_common.rs

```diff
@@ -1021,39 +1021,45 @@ rt_test! {
     // other tasks.
     #[test]
     fn ping_pong_saturation() {
+        use std::{mem::drop, sync::atomic::{Ordering, AtomicBool}};
```

Comment:
The prelude already has `drop`.
```suggestion
        use std::sync::atomic::{Ordering, AtomicBool};
```